doc/rst/source/explain_-n_full.rst_

```diff
@@ -22,7 +22,8 @@ The following modifiers are supported:
 - **+a** - Switch off antialiasing (where supported) [default uses antialiasing].
 - **+b** - Override boundary conditions used, by appending *g* for geographic, *p* for periodic, or *n* for
   natural boundary conditions. For the latter two you may append **x** or **y** to specify just one direction, otherwise
-  both are assumed.
+  both are assumed. For more information see :ref:<reference/file-formats:boundary-conditions-for-operations-on-grids>`.
```

Comment:
This doesn't work. 

I think you need to add a label in `reference/file-formats.rst`, like:
```
.. _grid-boundary-conditions
```
and reference it like:
```
:ref:`grid-boundary-conditions`
```

Reply:
I've just made a commit.